concent_api/utils/fields.py

```diff
@@ -0,0 +1,24 @@
+import base64
+
+from django.db import models
+
+
+class Base64Field(models.TextField):
+    """ Base64 encoding field for storing binary data in Django TextFields. """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.field_name = None
+
+    def contribute_to_class(self, cls, name, private_only=False, virtual_only=models.fields.NOT_PROVIDED):
+        if self.db_column is None:
+            self.db_column = name
+        self.field_name = name + '_base64'
```

Comment:
What's the purpose of this `_base64` suffix?

Looks to me like something analogous to `something` and `something_id` with foreign keys where the normal field stores the object (or makes a query) and the `_id` field has the raw `_id`. But I don't see it here. You're not storing both encoded and decoded version of the data here so this is not the same.

Reply:
It rather works like MoneyField, where you have general value stored with currency, and __amount which shows only float value.
In example:
provider_public_key gives b'\xe7M\xc8\x9d\xc2\x1di\x93\xdd\x19\xd1d\xea#i\xcb^\xfb\xc4\x89\xb7\xf8\xff\xc9_p\xae+\xa2t3\xfc\xf01\x05r\x8f\x96\x0b\x97\x16\xdc*SD^\x17\t(\xfc\xb8\x80\xb0\xc5=1\x99\x01\x92Z\xcf8z.'
provider_public_key_base64 gives '503IncIdaZPdGdFk6iNpy177xIm3+P/JX3CuK6J0M/zwMQVyj5YLlxbcKlNEXhcJKPy4gLDFPTGZAZJazzh6Lg=='

Purpose for this is that you can't query on binary data.